src/chi/utilities/display.scss

```diff
@@ -18,3 +18,21 @@
     }
   }
 }
+
+.-d-print--only {
+  display: none;
+}
+
+.-d-screen--only {
+  display: block;
+}
+
+@media print {
+  .-d-print--only {
+    display: block;
+  }
+
+  .-d-screen--only {
+    display: none;
+  }
+}
```

Comment:
`@ media print` looks good and I know it works in both cases, but I expected to see the same for `@ media screen`

Reply:
Ugh we don't, i thought i had deleted all instances from the file